core/src/main/scala/org/http4s/UrlForm.scala

```diff
@@ -1,30 +1,31 @@
 package org.http4s
 
 import cats._
+import cats.data._
 import cats.effect.Sync
 import cats.implicits.{catsSyntaxEither => _, _}
 import org.http4s.headers._
 import org.http4s.parser._
 import org.http4s.util._
 import scala.io.Codec
 
-class UrlForm private (val values: Map[String, Seq[String]]) extends AnyVal {
+class UrlForm private (val values: Map[String, Chain[String]]) extends AnyVal {
   override def toString: String = values.toString()
 
-  def get(key: String): Seq[String] =
-    this.getOrElse(key, Seq.empty[String])
+  def get(key: String): Chain[String] =
+    this.getOrElse(key, Chain.empty[String])
```

Comment:
I have zero clue how this works calling `this` rather than `values`

Reply:
Line 18

Reply:
Thank you, I feel more sane now.